packages/backend/src/handlers/actions/guild/saveGuildHandler.ts

```diff
@@ -23,17 +31,37 @@ export const saveGuildHandler = async (
   const { input, session_variables: sessionVariables } = req.body;
   const playerId = sessionVariables['x-hasura-user-id'];
 
+  const { guildInformation } = input;
+
+  let updatedGuild: GuildFragment;
   try {
-    const { guildInformation } = input;
-    await saveGuild(playerId, guildInformation as GuildInfoInput);
-    res.json({ success: true });
+    updatedGuild = await saveGuild(
+      playerId,
+      guildInformation as GuildInfoInput,
+    );
   } catch (error) {
     console.error(error);
     res.json({
       success: false,
       error: (error as Error).message,
     });
+    return;
```

Comment:
If the rest of the method below is moved into the `try`, you don't need this `return`.

```suggestion
```